etc/config.xml

```diff
@@ -3,8 +3,8 @@
         <payment>
             <unzer>
                 <active>0</active>
-                <public_key>s-pub-2a10ifVINFAjpQJ9qW8jBe5OJPBx6Gxa</public_key>
-                <private_key>s-priv-2a102ZMq3gV4I3zJ888J7RR6u75oqK3n</private_key>
+                <public_key></public_key>
+                <private_key></private_key>
```

Comment:
This does not match the commit message

Reply:
- Commit message adresses this change. 
-  Changelog is updated now.